### ISSUE TITLE
Your JS Fiddle example appears to be broken

### ISSUE THREAD

**Issue by rjhankison** [harttle/liquidjs#264]
Here's the link I'm trying to review: **https://jsfiddle.net/paycove/ha3709mL/6/**
Here's what I'm seeing when I try to load the page: **https://d.pr/i/NiMGKV**
It looks like this is causing no output to be sent to the HTML view: **https://d.pr/i/l8gYsa**

We're using our own JS fiddle to showcase our LiquidTemplating engine on our platform, so if you might give instructions on how we may update our JS fiddle to correctly use your package, that would be great!

**Comment by harttle** [comment]
> Your JS Fiddle example appears to be broken.

The filename/URL was changed and I forgot to update the demo :( Now it's updated to https://jsfiddle.net/pd4jhzLs/1/

> if you might give instructions on how we may update our JS fiddle

I'm not sure how you use RequireJS but your jsFiddle has some problems: 

1. `require([])` call is async, which means `liquid = require([])` won't work.
2. The line `engine.registerFilter('random', v => ` is not finished.

I've updated a vertion for you: https://jsfiddle.net/uxtfwkqL/1/ . BTW, you don't need RequireJS to use LiquidJS, it's also available on `window.liquidjs` in non-AMD environments.  Hope it helps!
